blog/_posts/2021-05-17-scala3-is-here.md

```diff
@@ -0,0 +1,424 @@
+
+To get a taste of all the new features that went into Scala 3, you can have a look at the [New in Scala 3 article](https://docs.scala-lang.org/scala3/new-in-scala3.html). For a streamlined and more thorough introduction, see the [Scala 3 Book](https://docs.scala-lang.org/scala3/book/introduction.html). You can try Scala 3 online without installing anything on your machine via [Scastie](https://scastie.scala-lang.org/?target=dotty), or you can follow the [Getting Started guide](https://docs.scala-lang.org/scala3/getting-started.html) to install it on your machine.
+
+One thing has changed completely in Scala 3 compared to Scala 2: it is macros. You can learn more about how they work in Scala 3 in the dedicated [documentation](https://docs.scala-lang.org/scala3/guides/macros/index.html).
+
+If you are a seasoned user of Scala 2, chances are you have some projects you'd like to port from Scala 2 to Scala 3. You may find the [Migration guide](https://docs.scala-lang.org/scala3/guides/migration/compatibility-intro.html#) helpful. It outlines the compatibility story between Scala 2 and Scala 3: source compatibility, binary compatibility, changed and dropped features, metaprogramming.
+
+And if you need any help in your Scala journey, you can find various resources to talk to fellow Scala users [here](https://www.scala-lang.org/community/).
+
+## Ecosystem
+
+Who's already on Scala 3? Which libraries can you already use? A great place to answer this question is [Scaladex](https://index.scala-lang.org/). Scaladex is an index of Scala libraries where you can explore the ecosystem by language version, platform or the type of job a library does. At the time of writing this article, there are 308 early adopter libraries on Scala 3, compared to 2597 Scala 2.13 libraries.
+
+## Releases and Guarantees in 3.x era
+
+We intend to continue releasing every 6 weeks after 3.0.0, bumping the patch version every time we do so. A stable 3.0.x release will be preceded with 3.0.x-RC1 release candidate 6 weeks prior to such a stable release. Such patch releases will contain bug fixes for bugs affecting the respective minor version. Patch versions will be forward and backward compatible with each other over the source, binary and TASTy compatibilities.
+
+Of course, we intend to continue the development of the language in ways other than bug fixing. New language features and standard library APIs will appear in future minor versions. As any addition in library APIs, these may break backward source compatibility in minor, rare ways. However, minor releases will not break backward binary or TASTy compatibility. Concretely, that means that libraries built with Scala 3.0.0 will keep working with Scala 3.x.y!
+
+
+## Library authors: Join our community build
+
+Scala 3 now has a set of widely-used community libraries that are built against every nightly Scala 3
+snapshot. Currently, this includes shapeless, ScalaPB, algebra, scalatest, scopt and squants.
```

Comment:
This list is looking pretty stale. Maybe just give a number instead?

Reply:
I counted 67 - https://github.com/lampepfl/dotty/tree/master/community-build/community-projects